modopt/tests/test_opt.py

```diff
@@ -21,94 +24,147 @@
     import_sklearn = True
 
 
-class dummy(object):
+dummy_func = lambda x_val: x_val
```

Comment:
Maybe specify that this is identity in the variable name

Reply:
renamed to `func_identity`